coderadar-test/src/main/asciidoc/restapi.adoc

```diff
@@ -722,6 +613,7 @@ include::{snippets}/metrics/tree/response-fields.adoc[]
 
 ===== Example Request
 
+// TODO still valid?
```

Comment:
Currently only get works, but both should be available. So you can leave the note for now.